src/app/home-page/home-page.component.html

```diff
@@ -4,6 +4,11 @@
 <ngx-json-ld [json]="websiteSchema"></ngx-json-ld>
 
 <ng-template #loggedInHomePage>
+  <div *ngIf="Dockstore.FEATURES.enableNewDashboard; else oldDashboard">
```

Comment:
I was thinking you would have this in home-logged-in.component.html, and then that component would display either the old or new dashboard. Or you introduce an intermediate component.